docs/_data/navigation.yml

```
@@ -17,3 +17,8 @@
   long_title: Support
   url: /support
   icon_class: fa fa-headset
+
+- title: Research
+  long_title: Research with Saros
```

```suggestion
  long_title: Research
```

'Research with Saros' sounds weird in my opinion. You could change it to 'Research Related to Saros', but I think just 'Research' is fine.